seatunnel-connectors-v2/connector-kudu/src/main/java/org/apache/seatunnel/connectors/seatunnel/kudu/source/KuduSource.java

```diff
@@ -175,7 +179,8 @@ public SeaTunnelRowType getSeaTunnelRowType(List<ColumnSchema> columnSchemaList)
 
         } catch (Exception e) {
             log.warn("get row type info exception", e);
```

Comment:
The same as above

Reply:
Thanks for your advice. I have fixed it!Thx